app/domain/authentication/authn_jwt/functions.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
```

Comment:
This is a good util functions that maybe can be used in other parts of conjur also and moving it out of the jwt authenticator will benefit our code base. I can see cases where we would like to concat paths in the future in other parts of the Conjur OSS.

Reply:
Absolutely, but I prefer to wait until there's a proven need for re-use before moving the file around to a new place.  That is, you might look at this, say, "Hey, this is general utility function, it doesn't belong here, it belongs on the root lib folder" or something similar.  But unless we already have a module  where it belongs, this usually ends up being a premature optimization, which has some costs.  Instead, I like to keep it at first close to where it's used.  Later, when we find ourselves _actually_ wanting to use it somewhere else, that's when I'd pull it out.